pkg/reconciler/ingress/resources/virtual_service.go

```diff
@@ -220,6 +220,11 @@ func makeVirtualServiceRoute(hosts sets.String, http *v1alpha1.HTTPIngressPath,
 			route.Retries.PerTryTimeout = types.DurationProto(http.Retries.PerTryTimeout.Duration)
 		}
 	}
+	if http.Retries == nil || http.Retries.Attempts == 0 {
```

Comment:
this is just `else` from he previous `if`.

Reply:
Thank you. Updated.